source/FPSciApp.cpp

```diff
@@ -436,7 +436,7 @@ void FPSciApp::updateControls(bool firstSession) {
 		rect = m_playerControls->rect();
 		removeWidget(m_playerControls);
 	}
-	m_playerControls = PlayerControls::create(*sessConfig, std::bind(&FPSciApp::exportScene, this), theme);
+	m_playerControls = PlayerControls::create(*sessConfig, std::bind(&FPSciApp::exportScene, this), theme   );
```

Comment:
Please try to avoid adding extra whitespace-based change. I suggest reviewing commits line by line in a tool like VS Code or w/ `git diff` to avoid committing changes like this.

Reply:
I'll keep this in mind. Thank you.